### DISCUSSION TITLE
Show SimpleLogin button on email input fields

### DISCUSSION THREAD

**Discussion by june8-o** [simple-login/app#1074] [Feature Requests]
While this option exists in the Firefox extension but not in the Safari extension.

**Comment by nguyenkims** [comment]
The current SimpleLogin safari extension was based on another technology and doesn't offer as much customization as Chrome/Firefox one. Recently Safari supports the same extension SDK as Chrome/Firefox, we have a plan to create a new Safari extension based on the current Chrome/Firefox code.
